sdk/core/Azure.Core/src/Shared/LruCache.cs

```diff
@@ -1,10 +1,12 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using System.Collections;
 using System.Collections.Generic;
 
-namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro
+#nullable disable
```

Comment:
Should we just add the nullable annotations and instead enable it?

Reply:
I removed this and now it just works... which is weird...

Reply:
It would need to be added so that it will compile when it is linked into the other libraries.

Reply:
I believe I added all the necessary annotations.

Reply:
Sorry I meant that we would need to add `#nullable enable` but it looks like we already did, so nevermind!